src/aggregate/aggregate_request.c

```diff
@@ -1210,6 +1210,45 @@ int buildOutputPipeline(AREQ *req, QueryError *status) {
   return REDISMODULE_ERR;
 }
 
+static void PushBefore(ResultProcessor *rp_to_place, ResultProcessor *upstream) {

```

Comment:
Add a comment explaining the usage.
```suggestion
static void PushUpStream(ResultProcessor *rp_to_place, ResultProcessor *rp) {

```